cmd/fleetctl/preview.go

```diff
@@ -117,13 +119,21 @@ This command will create a directory fleet-preview in the current working direct
 			fmt.Println("Fleet is now available at https://localhost:8412.")
 			fmt.Println("Username:", username)
 			fmt.Println("Password:", password)
-			fmt.Println("Note: You can safely ignore the browser warning \"Your connection is not private\". Click through this warning using the \"Advanced\" option.")
+			fmt.Println("Note: You can safely ignore the browser warning \"Your connection is not private\". Click through this warning using the \"Advanced\" option. Chrome users may need to configure chrome://flags/#allow-insecure-localhost.")
 
 			return nil
 		},
 	}
 }
 
+func previewDirectory() string {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		homeDir = "~"
+	}
+	return filepath.Join(homeDir, ".fleet", "preview")
```

Comment:
suggestion: It would be nice if this followed the XDG base directory specification (https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), putting the config into `$XDG_CONFIG_HOME/fleet/` instead. `$XDG_CONFIG_HOME` defaults to `$HOME/.config` if not overridden by the user.

It's really just nice to have, but avoids spreading config files across random parts of the filesystem.

Reply:
@nyanshak this is a good point. I'm tracking in https://github.com/fleetdm/fleet/issues/237 as the directory is not new to this PR and will require some work to migrate appropriately.